packages/react-devtools-shared/src/devtools/views/Profiler/CommitFlamegraph.js

```diff
@@ -96,9 +99,13 @@ type Props = {|
 |};
 
 function CommitFlamegraph({chartData, commitTree, height, width}: Props) {
-  const [hoveredFiberData, hoverFiber] = useState<number | null>(null);
+  const [hoveredFiberData, setHoveredFiberData] = useState<number | null>(null);
```

Comment:
This Flow type looks wrong. (Looks like it's been wrong for a while though FWIW- which is a bit concerning.)

We aren't storing a `number` in this state, but an object with `id` and `name` props (`TooltipFiberData`).